bluebottle/files/models.py

```diff
@@ -9,12 +9,27 @@
 
 from bluebottle.files.fields import ImageField
 from bluebottle.utils.models import AnonymizationMixin
+from bluebottle.utils.validators import FileMimetypeValidator
 
 
 class File(AnonymizationMixin, models.Model):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     created = models.DateField(_('created'), default=timezone.now)
-    file = models.FileField(_('file'), upload_to='files')
+    file = models.FileField(
+        _('file'),
+        upload_to='files',
+        validators=[FileMimetypeValidator(
+            allowed_mimetypes=[
```

Comment:
Can we user `settings.PRIVATE_FILE_ALLOWED_MIME_TYPES` here?

Reply:
Maybe. Is that list always the same? This includes zips, which might not be a good idea for private files.

Reply:
Oh no, this is another thing. Yeah, that sound like a good idea.